### PR TITLE
update ES image

### PR DESCRIPTION
**Pull request by QiaolingTang**
Log: 
arm64: http://pastebin.test.redhat.com/1085524
amd64: http://pastebin.test.redhat.com/1085517

### CODE REVIEW

**Comment by liangxia** [comment]
/approve

/hold
to let the feature team to decide when to get the fix in (by adding a comment `/unhold`).

File: features/step_definitions/logging.rb [hunk 1]
```diff
@@ -1078,13 +1078,11 @@
...
-  if version == "6.8"
-    # get the arch of node
-    @result = admin.cli_exec(:get, resource: "nodes", l: "kubernetes.io/os=linux", output: "jsonpath={.items[0].status.nodeInfo.architecture}")
-    # set xpack.ml.enable to false when testing ES 6.8 on arm64 cluster
-    if @result[:response] == "arm64"
-      cm_patch << ["p", "MACHINE_LEARNING=false"]
-    end
+  # get the arch of node
+  @result = admin.cli_exec(:get, resource: "nodes", l: "kubernetes.io/os=linux", output: "jsonpath={.items[0].status.nodeInfo.architecture}")
+  # set xpack.ml.enable to false when testing ES 6.8 on arm64 cluster
```
Comment (liangxia) [inline features/step_definitions/logging.rb:1083]: Looks like the comment needs update to make it in sync with the code changes.

Comment (QiaolingTang) [reply features/step_definitions/logging.rb:1083]: Yes, I'll update it. Thanks.

**Pull request merged**